### DISCUSSION TITLE
Author avatar on Author entry-meta

### DISCUSSION THREAD

**Discussion by balrogmedia** [bootscore/bootscore#785] [Q&A]
Hello,
I have installed Molongui in order to have a co-author:
![Screenshot 2024-05-29 at 12 05 40](https://github.com/bootscore/bootscore/assets/98542232/a12ecbc3-c4e8-4b3e-8012-b40e2204626b)
Is there a way with BS to not need to use a plugin to achieve the same result?

Also, I was wondering if there is a way to add the Author avatar like in this example:

![Screenshot 2024-05-29 at 12 02 29](https://github.com/bootscore/bootscore/assets/98542232/e6853762-2fee-423b-abff-81441d3902ab)

Thank you!

**Comment by crftwrk** [comment]
Hello,

> Is there a way with BS to not need to use a plugin to achieve the same result?

WordPress doesn't support more than one author on a post/page. So, you need a plugin to add more authors here.

> Also, I was wondering if there is a way to add the Author avatar like in this example

Yes, you can modify the author function in child's `functions.php` to show author avatar in meta: 

```php
/**
 * Author
 */
function bootscore_author() {
  $display_author = apply_filters('bootscore/meta/author', true);

  // Check if the filter returns false, if so, return early without displaying the author
  if (!$display_author) {
    return;
  }

  $author_id = get_the_author_meta('ID');
  $author_name = get_the_author();
  $author_url = esc_url(get_author_posts_url($author_id));
  
  // Adding custom classes to the avatar
  $avatar_args = array(
    'class' => 'custom-avatar rounded-circle border p-1' // Add your custom classes here
  );
  $author_avatar = get_avatar($author_id, 32, '', '', $avatar_args); // 32 is the size of the avatar

  $byline = sprintf(
    esc_html_x('by %s', 'post author', 'bootscore'),
    '<span class="author vcard">' . $author_avatar . ' <a class="url fn n" href="' . $author_url . '">' . esc_html($author_name) . '</a></span>'
  );

  echo '<span class="byline"> ' . $byline . '</span>'; // WPCS: XSS OK.
}
```

Solved?
